scalableminds/webknossos
# Tri-State plane display in 3D viewport

Add a new display mode for the planes in the 3D viewport which hides them alltogether. Also, move the setting to the 3D viewport where it can be more easily discovered.
Additionally, I added a setting to hide the dataset bounding box in the 3D view as requested in the issue.

![tri-state-plane-toggle-v3](https://user-images.githubusercontent.com/1702075/116593887-e111ff00-a921-11eb-94e4-4371f3432429.png)


TODO:
- [x] Add an evolution that modifies the saved user settings and recommended configurations. (@fm3 Could you point me to an evolution that modifies json in the postgres database?)

### URL of deployed dev instance (used for testing):
- https://tristateplanedisplay.webknossos.xyz

### Steps to test:
- Open a tracing and use the different plane display settings. The settings can be found when clicking the cogwheel in the upper right of the 3D viewport. The new option (left) should hide everything related to the planes.
- Test the setting to hide the dataset bounding box. The setting should be persisted.
- Create a task type and activate the recommended settings. There should be an explanation which values can be used for the `tdViewDisplayPlanes` setting and changing it to something else should not validate.

### Issues:
- fixes #5225
- fixes #3906

------
